app/models/dialog_field_button.rb

```diff
@@ -1,2 +1,3 @@
 class DialogFieldButton < DialogField
+  AUTOMATE_VALUE_FIELDS = %w(required read_only visible).freeze
```

Comment:
Honestly, I don't know why this file exists? I don't think there are any `DialogFieldButton`s being used today and I don't even think you can create them in the UI, much less make them dynamic. I think this line can go away and we can leave the class as is.